node-commons/src/main/java/org/apache/iotdb/commons/conf/CommonConfig.java

```diff
@@ -20,20 +20,27 @@
 
 import org.apache.iotdb.commons.client.property.ClientPoolProperty.DefaultProperty;
 import org.apache.iotdb.commons.cluster.NodeStatus;
+import org.apache.iotdb.commons.cluster.RegionStatus;
+import org.apache.iotdb.commons.consensus.DataRegionId;
+import org.apache.iotdb.commons.consensus.SchemaRegionId;
 import org.apache.iotdb.commons.enums.HandleSystemErrorStrategy;
 import org.apache.iotdb.tsfile.fileSystem.FSType;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.File;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.TimeUnit;
 
 public class CommonConfig {
 
   public static final String CONFIG_NAME = "iotdb-common.properties";
   private static final Logger logger = LoggerFactory.getLogger(CommonConfig.class);
 
+  private final Map<SchemaRegionId, RegionStatus> schemaRegionStatusMap = new ConcurrentHashMap<>();
+  private final Map<DataRegionId, RegionStatus> dataRegionStatusMap = new ConcurrentHashMap<>();
```

Comment:
Is it better to manage these two parameters on DataNodeRegionManager instead of CommonConfig? Since they are only maintained by DataNode, but both ConfigNode and DataNode will cache a CommonConfig instance.